frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -321,17 +321,10 @@ export const savedInsightsLogic = kea<savedInsightsLogicType>([
                   }
               ]
             | void => {
-            if (router.values.location.pathname === urls.savedInsights()) {
```

Comment:
Noticed we don't check the URL in other places, maybe should updated `urls.savedInsights()` as well? (then can keep this check)

Reply:
I think this line is here for instances when the `savedInsightsLogic` is mounted, but you are not actually on the saved insights page. We should probably adapt this line to still check the pathname and just ignore the url params there (url starts with or something).

Reply:
The cleanest way to do this check is to check `sceneLogic.values.activeScene`, like here: https://github.com/PostHog/posthog/blob/bab882c5fec15f9b6af72c252f52276ad4df774f/frontend/src/scenes/insights/insightSceneLogic.tsx#L235